data/news.yml

```
@@ -60,7 +60,7 @@
 
 - date: 2022 03 08
   headline: >
-    <a href="https://about.dandiarchive.org/2022/03/08/microscopy-workshop.html">
+    <a href="https://www.dandiarchive.org/2022/03/08/microscopy-workshop.html">
```

We recently changed the URLs of our sites.  Please see the details [here](https://github.com/dandi/dandi-about/issues/84#issuecomment-2581666275).
```suggestion
    <a href="https://about.dandiarchive.org/2022/03/08/microscopy-workshop.html">
```